backend/model/Submission.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose, { Schema, Document, Types } from 'mongoose';
+
+export interface ISubmission extends Document {
+  email: string;
+  submissionDate: Date;
```

Comment:
Do you mean `submissionTime` since we already know the date the user submit the test
```suggestion
  submissionTime: Date;
```